OpenRA.Mods.RA/Traits/PaletteEffects/LightPaletteRotator.cs

```diff
@@ -13,7 +13,7 @@
 using OpenRA.Graphics;
 using OpenRA.Traits;
 
-namespace OpenRA.Mods.Common.Traits
+namespace OpenRA.Mods.RA.Traits
```

Comment:
The trait is not used in the `cnc` mod. Never knew about lights on the Mammoth tank.